src/components/products-page/products-item-card.component.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import './products-item-card.styles.scss';
+
+import StandardButton from '../../components/standard-button/standard-button.component';
+
+const ProductsItemCard = (props) => (
+    <div className="card-background">
+        <div className="items-container">
+            <div className="card-heading">
+                ココナッツオイル in<br />
+                {props.cardHeading}
+            </div>
+            <img className="image" src={props.image} alt="img" />
+            <div className="textbox">{props.textbox}</div>
+            <StandardButton width="266px" height="28px">商品ページへ</StandardButton>
```

Comment:
color of the button doesn't match the design

Reply:
also might want to consider a pressed state color change for the button. like an ever so slightly darkened hover color.

Reply:
it shouldn't end up as a merge conflict as long as you haven't changed that same button in another branch.
you'll merge this feature branch into main, which will be an update to main. so theres no problem there.
for other feature branches, if they're not using the button then there wont be any problems at all.
if it is used, and assuming this branched is already merge into main, then you would jut have to pull to get the updated changes to the button from main.